### DISCUSSION TITLE
'Actual' License is GPL 2.0?

### DISCUSSION THREAD

**Discussion by mylemans** [PHPOffice/PhpSpreadsheet#2996] [General]
I noticed how PhpSpreadsheet uses mpdf and that is licensed under GPL 2.0 (see https://github.com/mpdf/mpdf/blob/development/LICENSE.txt)

Wouldn't it mean that this project isn't really usable as a MIT licensed work as you'll have to comply to the GPL license?

While the following is talking about GPL3 I think the same applies here:

https://opensource.stackexchange.com/questions/1640/if-im-using-a-gpl-3-library-in-my-project-can-i-license-my-project-under-mit-l

**Comment by MarkBaker** [comment]
PhpSpreadsheet ___can___ use mpdf, it's an option, not mandatory, and is not part of the PhpSpreadsheet project, or the distribution. The PhpSpreadsheet project is standalone and MIT Licensed. If you choose to use mpdf with PhpSpreadsheet that doesn't change or invalidate the PhpSpreadsheet MIT license in any way; but it may change the license for your own projects if you use both libraries.
